db/seeds.rb

```diff
@@ -20,6 +20,14 @@
 p "Generating random denounces for main user!"
 denounces = FactoryBot.create_list(:denounce, 10, user: user_test)
 
+p "Generaring random attachment for main user!"
+denounces.each do |denounce|
+  files = ['unsplash.jpeg', 'file.pdf'].take(rand(1..2))
```

Comment:
you can use [].sample, same idea

Reply:
An array is required and not a string so I think it could be `[].sample(rand(1..2))`